bin/lambda-cfn.js

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env node
+
+var cli = require('../lib/cli');
+var d3 = require('d3-queue');
+
+var parsed;
+try { parsed = cli.parse(process.argv.slice(2), process.env); }
+catch (err) { return finished(err); }
+
+// if (parsed.options.all) {
+  // call a helper function to compile the list of
+  // stacks for the command to loop over
+  //var q = d3.queue(1);
+  //for (var stack in parsed.options.stacks) {
+    // modify parsed and pass to command loop
+    // q.defer(cli.main, modified_parsed)
+    // q.await(function(err,data) {});
+//  }
+// } else {
+  cli.main(parsed, finished);
+
+  function finished(err, data) {
+    if (err) process.stderr.write(err.message + '\n');
```

Comment:
Looks like we're not using any linters (https://github.com/mapbox/lambda-cfn/blob/master/package.json#L31) despite having configurations for both jscs and jshint in the package.json for this project (this should be another issue that we fix in another PR). Also not seeing any configs to stopping or catching the use of `console.log` so this must be inherited from cfn-config as you suggested.